src/components/main-menu/navbar-menu.tsx

```diff
@@ -14,15 +15,15 @@ import {
 
 interface Props {
   handleClose: () => void;
+  soundOn: boolean;
+  setSoundOn: (value: boolean) => void;
 }
 
-export const Header: FC<Props> = ({ handleClose }) => {
+export const NavbarMenu: FC<Props> = ({ handleClose, soundOn, setSoundOn }) => {
   const router = useRouter();
-  // TODO: Implement sound hook
-  const [sound, setSound] = useState(true);
 
   const onSoundClick = () => {
```

Comment:
Should we call this toggleSound, or you want to have the same naming as the hook in the component?

Reply:
if you are referring to the `setSound` prop, "toggle" wouldn't apply since the fn takes a boolean as input